meta/engineering_principles.md

```diff
@@ -0,0 +1,137 @@
+get large enough for their stewardship to become burdensome.
+
+- Examples:
+
+  - artsy/meta
+  - artsy/force
+  - artsy/eigen
+  - artsy/emission
+
+- Further Reading:
+
+  - https://artsy.github.io/series/open-source-by-default/
+  - https://ashfurrow.com/blog/open-source-ideology/
+  - https://code.dblock.org/2015/02/09/becoming-open-source-by-default.html
+  - https://www.objc.io/issues/22-scale/artsy/
+
+### Own Your Dependencies
+
+Take the best dependencies to fit your problem and make them better. Rather than try re-invent the wheel, give time
+to take the best dependencies and make them fit into our use cases. We’re not the biggest team, so the most value
+for someone's time can be achieved by writing the glue code between larger projects.
+
+Consider all of the pieces related to your app as being part of your work. E.g. just like you would consider
+working on an API when you're doing front-end work, making sure that React or TypeScript behave like we'd want is
+an important part of product work. It's another part of our stack.
+
+Aim to be a trusted contributor to the communities surrounding your work, communicate clearly, publicly and be
+empathetic to the priorities of others.
+
+- Examples:
+
+  - Grape
+  - CocoaPods
+  - The Artsy Omakase
+
+### Incremental Revolution
+
+Introduce new technologies slowly and incrementally. Avoid re-writes. Build tools to allow hybrids of different
+types of technology when possible. Sometimes you need to make a big leap, but aim to approach them incrementally.
+
+Explore bleeding-edge technologies on projects with an end-date and can become safely classed "done". These can be
+used to inform decisions on long-running projects. Run spike projects when trying to settle between technology
+trade-offs.
+
+- Examples
+
+  - Emission / Reaction
+  - Stitch
+  - Metaphysics
+
+- Further Reading
+
+  - Building a Pod Library http://artsy.github.io/blog/2018/04/17/making-a-components-pod/
+  - http://artsy.github.io/blog/2017/09/05/Modernizing-Force/
+
+### Minimal Viable Process
+
+As a team grows, so does the need for processes. Find ways to add process without additional work. Build automated
+tools to help others. Use systems like RFCs to broaden the availability of information on process changes.
+
+For meetings over 2-3 people, provide agendas and make as many people optional as possible. We've seen great
+success on automation via documentation for recurring meetings.
+
+- Examples
+
+  - Danger
+  - Peril
+  - RFC Process
+  - artsy/meta
+
+- Further Reading
+
+  - http://artsy.github.io/blog/2018/05/25/support-process/
+  - http://artsy.github.io/blog/2017/09/04/Introducing-Peril/
+  - http://artsy.github.io/blog/2018/02/02/artsy-apogee/
+
+### Scale your Impact
+
+We built an outsized reputation for our size as an engineering team via the above principals. This gives the
+ability to move local impact to larger impact.
```

Comment:
I really relate to this kind of "everyone taking responsibility for the whole group and product" message. We're a lean team so we must maximize the leverage of our individual efforts by sharing openly, etc. I actually think "leverage" is a nice meaningful term that could be incorporated into the title.